### PR TITLE
Add "Parent" and "Included Modules" to sidebar

### LINKED ISSUE #288: `Parent` and `Included Modules` in side menu
Hello!

Please, add `Parent` and `Included Modules` sections in side menu alongside methods.

Reference: https://ruby-doc.org/core-2.7.0/Array.html

---

image

---

image

### PR DESCRIPTION
**Pull request by AlexWayfer**
Resolve #288

### CODE REVIEW

File: app/models/ruby_object.rb [hunk 1]
```diff
@@ -56,6 +56,20 @@ def ruby_methods
     @ruby_methods ||= body[:methods].collect { |m| RubyMethod.new(m) }
   end
 
+  def superclass
+    return @superclass if defined?(@superclass)
+
+    value = body[__method__]
```
Comment (natematykiewicz) [inline app/models/ruby_object.rb:62]: Is there a reason you used `__method__` here (and in `included_modules`)? This is the first time I've ever seen this. 

Based on the docs I just read, it seems like you could have just used a symbol literal of `:superclass`. Is `__method__` better for some reason?

Comment (AlexWayfer) [reply app/models/ruby_object.rb:62]: > Is there a reason you used `__method__` here (and in `included_modules`)?

DRY.

> Based on the docs I just read, it seems like you could have just used a symbol literal of `:superclass`. Is `__method__` better for some reason?

Yes, I could. Yes, it's better for DRY reason.

**Comment by AlexWayfer** [comment]
> It seems like there should be some tests for this. Perhaps something in `test/models/ruby_method_test.rb`?

Thank you, added. With new `RubyObject#==` method (I'm not sure about this).

**Comment by colby-swandale** [comment]
This is great! Thank you so much and apologies for taking a while to respond.

**Comment by natematykiewicz** [comment]
I just saw that this got merged and it's really awesome! I've been trying to use Ruby API as my main ruby documentation tool, but that's been hard without this feature (and a few other smaller ones). This is a big deal to me. Thanks!

**Pull request merged**